qiskit/quantum_info/synthesis/clifford_decompose.py

```diff
@@ -102,8 +105,8 @@ def decompose_clifford_bm(clifford):
     ret_circ = QuantumCircuit(num_qubits, name=clifford_name)
     for qubit in range(num_qubits):
         pos = [qubit, qubit + num_qubits]
-        table = clifford.table.array[pos][:, pos]
-        phase = clifford.table.phase[pos]
+        table = clifford.tableau()[pos][:, pos]
+        phase = clifford.paulis.phase[pos]
```

Comment:
We should move the initial calculation of `table` and `phase` outside the loop, but the indexing obviously needs to stay inside.